opm/flowdiagnostics/utility/AssembledConnections.cpp

```diff
@@ -256,9 +256,7 @@ CSR::accumulateRowEntries(const int               numRows,
     }
 
     // Note index range: 1..numRows inclusive.
-    for (decltype(this->ia_.size())
-             i = 1, numRows = this->ia_.size() - 1;
-         i <= numRows; ++i)
+    for (int i = 1; i <= numRows; ++i)
```

Comment:
I see zero effect on performance from this, in my informal tests (on arrays of 100 million elements), so I don't think that type conversion is a problem. Do you suggest that whenever using `std::vector<>::operator[]()` we should use only the `size_type`? I think that is quite restrictive, unnecessary (from a performance p.o.v.) and less readable (that is my main reason to prefer the changed version).